s3/templates/cloudfront_oai.json.tpl

```diff
@@ -23,6 +23,27 @@
       "Action": "s3:GetObject",
       "Resource": "arn:aws:s3:::tdr-upload-files-cloudfront-dirty-${environment}/*"
     },
+    {
```

Comment:
Are there changes in place for the OAC? If so should probably rename this file to something more generic.